hashring/hashring.go

```diff
@@ -76,6 +80,7 @@ func New(hashfunc func([]byte) uint32, replicaPoints int) *HashRing {
 		hashfunc: func(str string) int {
 			return int(hashfunc([]byte(str)))
 		},
+		logger: logging.Logger("ring"),
```

Comment:
@thanodnl +1 maybe something for the future.

When you say "reason about which member logged the line", do you mean, it's easier for a human to see which node it's from given a stream of logs from all nodes?

I would have hoped that log analysis (ELK?) services are able to tell where logs came from.

But you're right, if we do need to log this, I think we should do it in one place. Probably at the root-logger level.